.github/workflows/build.yml

```
@@ -116,7 +128,14 @@
         if [ -n "${{ inputs.version }}" ]; then
           VERSION_ARG="-Pversion=${{ inputs.version }}"
         fi
-        ./gradlew --no-daemon --no-configuration-cache --scan nativeCompile $VERSION_ARG
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)